speech-therapy/index.html

```diff
@@ -49,8 +49,8 @@ <h2>Session 1: 7/3/2024 30min</h2>
          <p>Takeaway: Once again, I find speech therapy incredibly helpful. As I become more mindful of my social interactions and emotions, I am starting to recognize my coping mechanisms. In the past, I read books on how to make friends and gain others' trust, but they mainly focused on following rules and techniques. In recent years, to cope with stress, anxiety, and low self-esteem, I stopped listening to both myself and others. My communication was driven by my own insecurity.</p>
           <p>I remember watching the TED talk below more than 10 times since 2016. It is easier said than done to have good conversations. I did the opposite of its advice, especially in the past few years: I was self-centered, talked to impress, spoke excessively, and showed little interest in others. As I become more at peace with my emotions, I am beginning to pay more attention to both the other person and myself. Did I respond just to please others? Did I fully understand what the other person wanted to say? Now, I am still learning more about my communication patterns, but I notice my listening skills have much improved.</p>
 
-          <iframe width="640" height="360" src="https://youtube.com/embed/R1vskiVDwl4?si=nxJ95PAEKabcjMGq;showinfo=0" frameborder="0" allowfullscreen></iframe>
-    </ul>
-        
+      </ul>
+          <iframe width="640" height="360" src="https://youtu.be/R1vskiVDwl4?si=nxJ95PAEKabcjMGq;showinfo=0" frameborder="0" allowfullscreen></iframe>
```

Comment:
It looks like when you merged the Pull Request, you choose the old version of the URL instead of the new one.
old: `https://youtu.be`
new: `https://youtube.com/embed`